liquidsoap.systemd.in

```diff
@@ -1,16 +1,16 @@
 [Unit]
-Description=Liquidsoap daemon
+Description=@script_name@ Liquidsoap daemon
 After=network.target
 Documentation=http://liquidsoap.fm/
 
 [Service]
 Type=forking
 User=@user@
-PIDFile=@pid_dir@/run.pid
+PIDFile=@pid_file@
 WorkingDirectory=@base_dir@
 ExecStart=@liquidsoap_binary@ @run_script@
-Restart=always
+#Restart=always
```

Comment:
Is this intended?

Reply:
Oops. That was when I was debugging something. Will correct, squash and update this branch today.